### PR TITLE
Feat/zig 2324 adapt fe after cleaned login signup responses

### PR DESCRIPTION
**Pull request by muradmalik777**

### CODE REVIEW

File: src/store/actions/user.js [hunk 1]
```diff
@@ -151,21 +151,31 @@ export const getDailyUserBalance = (payload) => {
...
-        type: GET_USER_DATA,
-        payload: responseData,
-      };
+      if (token) {
+        const payload = {
+          token: token,
+        };
+        const responseData = await tradeApi.userDataGet(payload);
+        const action = {
+          type: GET_USER_DATA,
```
Comment (cwagner22) [inline src/store/actions/user.js:169]: It would be clearer with SET_USER_DATA

**Comment by cwagner22** [comment]
I true that getUserData will be called twice when logging in/signing up. We can discuss how to avoid that.

File: src/store/actions/user.js (hunk 1 above)
Comment (muradmalik777) [reply src/store/actions/user.js:169]: Done

File: src/store/initialState.js [hunk 2]
```diff
@@ -41,6 +41,7 @@
  * @typedef {Object} DefaultSessionData
  * @property {string} status
  * @property {number} validUntil
+ * @property {'login'|'signup'|''} initEvent
```
Comment (cwagner22) [inline src/store/initialState.js:44]: Is that needed?

**Pull request merged**